packages/react/src/components/Slider/Slider.js

```diff
@@ -445,7 +445,10 @@ export default class Slider extends PureComponent {
     const { value } = evt.target;
 
     this.setState({ isValid: validity });
-    this.props.onBlur({ value });
+
```

Comment:
Since `onBlur` is a function in proptypes, we don't need to check whether or not it is a function here, just whether or not it's defined
```suggestion
if (this.props.onBlur) {
  this.props.onBlur({ value });
}
```